examples/workflows/resource_delete_with_flags.py

```diff
@@ -0,0 +1,41 @@
+from hera.workflows.models import Inputs, Parameter
+from hera.workflows.resource import Resource
+from hera.workflows.steps import Step, Steps
+from hera.workflows.workflow import Workflow
+
+create_configmap = Resource(
+    name="create-configmap",
+    action="create",
+    manifest="""apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: resource-delete-with-flags
+  labels:
+    cleanup: "true"
+data:
+  key: value
+""",
+)
+delete_resource = Resource(
+    name="delete-resource",
+    action="delete",
+    flags=["configmap", "--selector", "{{inputs.parameters.selector}}"],
+    inputs=Inputs(
+        parameters=[Parameter(name="selector")],
+    ),
+)
+
+with Workflow(
```

Comment:
Given that the templates are in order, should we just use the context handler to define all the templates inside the workflow instead of having to add them?